content/workload-reduction-toolkit/address-workload-issues/feedback-and-marking/review-marking-practice-with-an-impact-matrix.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Review marking practice with an impact matrix
+layout: /markdown_resource.html.erb
+---
+
+# Review marking practice with an impact matrix
+
+<strong class="govuk-tag">Template</strong>
+
+{inset-text}
+
+## School details
+
+**School name:** The City of Norwich School
+
+**Location:** Norwich
+
+**Phase:** Secondary
+
+**Number of pupils:** 1550
+
+{/inset-text}
+
+<div class="govuk-grid-row dfe-width-container">
+  <div class="govuk-grid-column-full">
+    <div class="info-box">
+      <div class="info-box__corner">
+        <img src="/assets/images/bullseye.svg" alt="Bullseye icon">
+      </div>
+      <h2 class="govuk-heading-m">
+        Impact and outcomes
+      </h2>
+      <p>
+        <ul>
+          Using the impact matrix to review marking practices at the City of Norwich School has meant that: 
+          <li>homework activities have been simplified and marking criteria removed</li> 
```

Comment:
The intro sentence needs to be shifted outside the `<ul>` otherwise it'll look a little funky!

```suggestion
      <p>
         Using the impact matrix to review marking practices at the City of Norwich School has meant that: 
        <ul>
          <li>homework activities have been simplified and marking criteria removed</li> 
```

<img width="571" alt="Screenshot 2024-02-20 at 10 26 32" src="https://github.com/DFE-Digital/improve-school-workload-and-wellbeing/assets/18436946/65fe3268-6b00-4058-b7d6-a6534b63b771">